src/components/Write/Write.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {useRef, useState} from 'react';
+import Keyboard from "react-simple-keyboard"
+import "react-simple-keyboard/build/css/index.css";
+
+type Props = {
+}
+export default function Write(props: Props) {
+const keyboardRef = useRef(null)
+    const [input, setInput] = useState("")
+    const [layout, setLayout] = useState("default")
+
+    function onChange(input: any) {
+        setInput(input);
+    }
+
+    function onKeyPress(button: any) {
```

Comment:
```suggestion
    function onKeyPress(button: KeyboardEvent) {
```
not 100% sure about this one